test/cpp/test_aten_xla_tensor.cpp

```diff
@@ -5448,12 +5448,13 @@ TEST_F(AtenXlaTensorTest, TestIndexAdd) {
     }
   }
   ExpectCounterNotChanged("aten::.*", cpp_test::GetIgnoredCounters());
-  ExpectCounterChanged("xla::index_add_", cpp_test::GetIgnoredCounters());
+  ExpectCounterChanged("xla::index_add", cpp_test::GetIgnoredCounters());
```

Comment:
Hmm, this is mixing the stuff in my pr in this pr. I guess it is due to you try to merge to master directly. This will make review a bit tricky

Reply:
I wonder if a rebase of `master` can help simplify things here?

Reply:
@JackCaoG if you are planning on merging your PR, we can wait until the landing. It automatically resolve, an alternative would be to create a PR against your branch as well. WDYT?

Reply:
you can wait when my pr merged.